content/guides/embed/ui-elements/explorer-metadata.md

```diff
@@ -0,0 +1,232 @@
+    const fieldsToShow = [
+        // Determine if the user can edit the metadata directly from Content Explorer component
+        { key: `${metadataSourceFieldName}.name`, canEdit: false },
+
+        // Determine label alias on metadata column with displayName prop
+        { key: `${metadataSourceFieldName}.industry`, canEdit: false, displayName: "alias" },
+        { key: `${metadataSourceFieldName}.last_contacted_at`, canEdit: true },
+        { key: `${metadataSourceFieldName}.role`, canEdit: true },
+    ];
+
+    // defaultView - a required prop to paint the metadata view.
+    // If not provided, you'll get regular folder view.
+    const defaultView = "metadata";
+
+    return (
+        <IntlProvider locale="en">
+            <div className="App">
+                <header className="App-header">
+                    <h2>Metadata view in Content Explorer</h2>
+                </header>
+                <section>
+                    <div className="metadata-based-view">
+                        <ContentExplorer
+                            rootFolderId={rootFolderID}
+                            token={token}
+                            metadataQuery={metadataQuery}
+                            fieldsToShow={fieldsToShow}
+                            defaultView={defaultView}
+                        />
+                    </div>
+                </section>
+            </div>
+        </IntlProvider>
+    );
+}
+
+export default App;
+```
+
+## Display name and key parameters
+
+* The `displayName` parameter is the display name of the template.
+* The `templateKey` parameter is a unique identifier of the template. It needs 
+to be unique across the enterprise for which you create the metadata template.
+If you don't provide the `templateKey` parameter, API creates a unique one
+based on the value in `displayName`.
+* The `[fields].displayName` parameter is the display name of the field as it
+is shown to the user in the web and mobile apps.
+* The `[fields].key` parameter is a unique identifier for a specific field in
+the template. The identifier must be unique within the template to which it
+belongs.
+
+## Key
+
+To decide which fields to show, the metadata Content Explorer uses metadata
+[field keys][field-key], not the [display names][display-name]. You can see the
+display names in the Admin Console and user view, but you can obtain the field
+keys through the API.
+
+The field keys are not changing, even if you change the metadata display name.
+This ensures that the functionality works properly, despite any changes to the
+metadata in UI view.
+
+### Key sanitization
+
+[Keys][field-key] are restricted to alphanumeric characters only:
+
+* No hyphens `-` or underscores `_` are permitted.
+* Only letters (`a-z, A-Z`) and numbers (`0-9`) are allowed.
+
+**Non-Latin characters:**
+
+If keys contain characters from non-Latin alphabets (such as Cyrillic, Arabic,
```

Comment:
the key is based on the display name, so it the display name contains non-Latin letters the key will be renamed.